paws_collector.js

```diff
@@ -440,6 +440,38 @@ class PawsCollector extends AlAwsCollector {
         this.reportDDMetric('api_throttling', 1)
         return cloudwatch.putMetricData(params, callback);
     };
+
+    /**
+     * Report the client errors and show case on DDMetrics and cloudwatch
+     * @param callback 
+     * @param errorCode - error type
+     */
+    reportClientErrors(errorCode = 'unknown', callback) {
```

Comment:
given that we're reporting a single error metric, perhaps `reportClientError` is a better name here.

Also, I think just passing a string here limits us in what clients are able to report. Using the `Error` object in Node might be a better interface for this as we can pull code from there: https://nodejs.org/api/errors.html#errors_error_code

@kkuzmin  thoughts?

Reply:
+1 for the object interfaces

Reply:
updated to error object from string